azure-pipelines.yml

```diff
@@ -106,10 +106,11 @@ stages:
           clean: true
 
         # Authenticate with service connections to be able to publish packages to external nuget feeds.
+        # Temporarily add devdiv-vssdk-unifiedPIAs-Service-Connection for an internal feed
         - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
           - task: NuGetAuthenticate@0
             inputs:
-              nuGetServiceConnections: azure-public/vssdk
+              nuGetServiceConnections: azure-public/vssdk, devdiv-vssdk-unifiedPIAs-Service-Connection
```

Comment:
Added service connection for internal feed. Note that this is only used for dnceng/internal pipeline, so PR build will still fail.